config.py

```diff
@@ -141,12 +154,14 @@ class TestConfig(Config):
 
 
 def db_generator(config):
-    db_type = config.DB_TYPE
-    if db_type == "mssql":
-        return MSSQL()
-    elif db_type == "postgres":
-        return PostgreSQL()
-    elif db_type == "sqlite":
-        return SQLite(path=config.SQLITE_FILE)
-    else:
-        raise Exception()
+    env_config = {
+        "dialect": config.DB_DIALECT,
+        "user": config.DB_USER,
+        "pwd": config.DB_PWD,
+        "server": config.DB_SERVER,
+        "port": config.DB_PORT,
+        "db": config.DB,
```

Comment:
We should probably rename this to something a little more clear — `db_name` (and `DB_NAME`) perhaps? Even then we probably need a comment in the README explaining what it is.

Reply:
I tried to keep the params consistent with the sqlsorcery library for compatibility reasons.